.github/workflows/manual-go-test.yml

```diff
@@ -0,0 +1,42 @@
+name: Manual Go Test Run
+
+on:
+  workflow_dispatch:
+
+# When a new revision is pushed to a PR, cancel all in-progress CI runs for that
+# PR. See https://docs.github.com/en/actions/using-jobs/using-concurrency
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
Ooooo! Ok, this is super fancy! I'm stealing this for use at my day job. Thanks!